usaspending_api/references/v2/views/filter_trees/filter_tree.py

```diff
@@ -0,0 +1,63 @@
+from abc import ABCMeta, abstractmethod
+from dataclasses import dataclass
+
+DEFAULT_CHILDREN = 0
+
+
+@dataclass
+class Node:
+    id: str
+    ancestors: list
```

Comment:
I mentioned this in person, but I find it very odd to have a tree where each node has a complete list of its ancestors. I feel like it would make more sense to either have just the parent, and traverse the tree from child to parent; or to not have the parent/ancestors at all and traverse from parent to child. If having the ancestor list works that's fine, it just seems more complicated than necessary.

Reply:
It might end up being removable, depending on how the many refactors of the front end pan out, but if we end up doing TAS similarly to how NAICS is being done as of time of writing, we need a reliable way to confirm if one node is ancestor of another, even if the nodes aren't provided in the same tree. Given that I'm not sure it will be removed and it was relatively easy to build, I was intending to start with the higher amount of requirements and pull it off if it's not necessary.

Reply:
> we need a reliable way to confirm if one node is ancestor of another

Can we just provide a `parent` or `parent_node` or `parent_tas` attribute back for the requested node that allows the answer to the question: _"Do you have a parent? Who is your parent"_?

Reply:
To be honest, I forget if that would suffice or not. I'll check in with Max now that we have one working filter tree and see what logic he's using at this point (remember that full ancestry like this is an inherent feature of NAICS)

Reply:
Will this still give you enough information to use the rest  of the API? In  order  to be as RESTful as possible and avoid possible ambiguity, the other endpoints introduced in this PR require the full path down  to the level you're searching for  children of.